common/djangoapps/student/views.py

```diff
@@ -181,33 +179,42 @@ def process_survey_link(survey_link, user):
     return survey_link.format(UNIQUE_ID=unique_id_for_user(user))
 
 
-def cert_info(user, course, course_mode):
+def cert_info(user, course_overview, course_mode):
     """
     Get the certificate info needed to render the dashboard section for the given
-    student and course.  Returns a dictionary with keys:
-
-    'status': one of 'generating', 'ready', 'notpassing', 'processing', 'restricted'
-    'show_download_url': bool
-    'download_url': url, only present if show_download_url is True
-    'show_disabled_download_button': bool -- true if state is 'generating'
-    'show_survey_button': bool
-    'survey_url': url, only if show_survey_button is True
-    'grade': if status is not 'processing'
+    student and course.
+
+    Arguments:
+        user (User): A user.
+        course_overview (CourseOverview): A course.
+        course_mode (str): The enrollment mode (honor, verified, audit, etc.)
+
+    Returns:
+        dict: A dictionary with keys:
```

Comment:
What I have there is the style that I've used throughout my PR, and what's used throughout lots of LMS. It's also what's recommended in the [Google style guide](https://google-styleguide.googlecode.com/svn/trunk/pyguide.html#Comments)